h2o-py/h2o/h2o.py

```diff
@@ -320,7 +323,7 @@ def upload_file(path, destination_frame=None, header=0, sep=None, col_names=None
     return H2OFrame()._upload_parse(path, destination_frame, header, sep, col_names, col_types, na_strings)
 
 
-def import_file(path=None, destination_frame=None, parse=True, header=0, sep=None, col_names=None, col_types=None,
+def import_file(path=None, pattern="", destination_frame=None, parse=True, header=0, sep=None, col_names=None, col_types=None,
```

Comment:
Right, it is based on semantics of our import - which can import multiple files by default multiple files (e.g, `import_file /data/iris` - can mean import single file, but also directory of files). 

But you have good point, @navdeep-G can you please summarize change in `api` channel. We can always deduct pattern as @st-pasha is suggesting - i like that solution.

Furthermore, I just realized that this is a breaking change in Python API.